quickstart/astro-cli.md

```diff
@@ -0,0 +1,57 @@
+## Prerequisites
```

Comment:
Maybe explicitly clarify that you don't need to be an Astronomer customer for this too